### ISSUE TITLE
ColumnToggler: XSS caused by script tags in headerText of Column

### ISSUE THREAD

**Issue by torbenw** [primefaces/primefaces#4913]
## 1) Environment
- PrimeFaces version: 6.2.19
- Application server + version: Apache Tomcat 8.5.13
- Affected browsers: Internet Explorer 11, Firefox 67.0.3, Chrome 75

## 2) Expected behavior
The headerText should be treated as text in the ColumnToggler. HTML tags and code should be escaped. The headerText is already properly escaped in the DataTable's header.

## 3) Actual behavior
A script tag in the headerText of a Column is executed when a ColumnToggler is attached to the DataTable and the page is loaded. The headerText is not escaped in the ColumnToggler.

## 4) Steps to reproduce
1. Open the provided XHTML example page.
2. An alert "xss" will be displayed.

## 5) Sample XHTML
```XHTML
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:h="http://java.sun.com/jsf/html" xmlns:f="http://java.sun.com/jsf/core" xmlns:p="http://primefaces.org/ui">
  <h:head />
  <h:body>
    <p:dataTable id="table">
      <f:facet name="header">
        <p:commandButton id="toggler" type="button" value="Toggler" />
        <p:columnToggler datasource="table" trigger="toggler" />
      </f:facet>
      <p:column headerText="&lt;script&gt;alert('xss')&lt;/script&gt;" />
    </p:dataTable>
  </h:body>
</html>
```

## 6) Sample bean
Not needed.

**Comment by melloware** [comment]
I will fix this. @Rapster can you add the security label please?

**Comment by ttsiebzehntt** [comment]
@tandraschko Are you going to fix this in 6.2.x? If yes, when can we expect a fix?

**Comment by melloware** [comment]
that would be @mertsincan who controls the Elite releases.

**Comment by ttsiebzehntt** [comment]
@mertsincan Any chance to get this into 6.2.22?

**Comment by mertsincan** [comment]
Fixed for 6.2.22 and 7.0.5
